lib/org/chromium/webidl/ast/CallbackInterface.js

```diff
@@ -12,14 +12,20 @@ foam.CLASS({
     {
       class: 'FObjectProperty',
       of: 'org.chromium.webidl.ast.Interface',
-      name: 'interface',
+      name: 'interfaceDetail',
     },
+    {
+      name: 'id',
+      expression: function(interfaceDetail) {
+        return interfaceDetail.id;
```

Comment:
`interface` cannot be used as a variable as we are running in strict mode and `interface` is a reserved word.
We could just do `ids: [ 'interface' ]` but I'm not certain if comparing interface objects would be efficient